static/js/components/SkipFinancialAidDialog.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import Button from 'react-mdl/lib/Button';
+
+const skipActions = (cancel, skip) => (
+  <div className="actions">
+    <Button
+      type='button'
+      className="dialog-button cancel-button"
+      onClick={cancel}>
+      Cancel
+    </Button>
+    <Button
+      type='button'
+      className="dialog-button save-button"
+      onClick={skip}>
+      Pay Full Price
```

Comment:
On my laptop this is rendering as `Pay Full` for some reason, can't figure it out:

![screenshot from 2016-10-05 16-51-57](https://cloud.githubusercontent.com/assets/863262/19131116/1eab199c-8b1c-11e6-9f45-759916b213d0.png)